query-node/joystream-query-node/mappings/members.ts

```diff
@@ -1,3 +1,32 @@
-export function handleMemberRegistered() {
-	console.log(handleMemberRegistered.name);
+import { Memberships } from '../generated/indexer/entities/Memberships';
+import { DB } from '../generated/indexer';
+
+export async function handleMemberRegistered(db: DB) {
+  // Get event data
+  const { AccountId, MemberId } = db.event.event_params;
+
+  let member = new Memberships({ accountId: AccountId.toString(), memberId: +MemberId });
+
+  // Save to database.
+  db.save<Memberships>(member);
+
+  // Query from database
+  member = await db.get(Memberships, { where: { memberId: MemberId } });
```

Comment:
Can we drop this from the example, as one would not want to do this in a normal mapping.

Reply:
Done in https://github.com/Joystream/joystream/pull/418/commits/e35594ca09ca47d7a2eb93d8e7e4b6a1f8c21b91